<?xml version="1.0"?>
<launch>
    <rosparam file="$(find baxter_hrc)/config/params.yaml" command="load"/>
    
    <node name="server_endpoint" pkg="baxter_hrc" type="server_endpoint.py" args="--wait" output="screen" respawn="true" />
    
    <node name="joint_states_service" pkg="baxter_hrc" type="joint_states_service.py" output="screen" />
    
    <group ns="left_group">
	  <node name="motion_planner_left" pkg="baxter_hrc" type="motion_planner.py" args="--limb left --offset 0.155 --obstacle 0.6" output="screen"  />
	  <include file="$(find baxter_hrc)/launch/demo.launch" />
    </group>
    
    <group ns="right_group">
	  <node name="motion_planner_right" pkg="baxter_hrc" type="motion_planner.py" args="--limb right --offset 0.15 --obstacle 0.65" output="screen"  />
	  <include file="$(find baxter_hrc)/launch/demo.launch" />
    </group>
    
    <node name="world_to_base" pkg="tf" type="static_transform_publisher" args="0 0 0.92 0 0 0 world base 10" />
    
    <node name="plan_manager" pkg="baxter_hrc" type="plan_manager.py" output="screen" />
</launch>
